src/common/packager.ts

```diff
@@ -192,7 +194,7 @@ export class Packager {
                                 args = args.concat("--resetCache");
                             }
                             if (runAs === PackagerRunAs.EXPONENT) {
-                                args = args.concat(["--root", ".vscode"]);
+                                args = args.concat(["--root", path.relative(this.projectPath, path.resolve(this.workspacePath, ".vscode"))]);
```

Comment:
This path needs to point to wherever the `exponentIndex.js` file is I believe, otherwise the exponent packager won't find the entry point (or the rest of the app)